plugins/module_utils/network/sonic/argspec/bgp_neighbors/bgp_neighbors.py

```diff
@@ -138,7 +138,8 @@ def __init__(self, **kwargs):
                                         'activate': {'type': 'bool'},
                                         'afi': {
                                             'choices': ['ipv4', 'ipv6', 'l2vpn'],
-                                            'type': 'str'
+                                            'type': 'str',
+                                            'required': True
```

Comment:
safi is also required.

Reply:
Santhosh has pointed out that because there is a "'required_together': [['afi', 'safi']]," directive present for the containing dictionary, it is sufficient to specify 'required' only for the 'afi' option.